uninstall.sh

```diff
@@ -1,18 +1,29 @@
 #!/bin/bash
-# This script removes brew and artifacts installed by it.
-# This is used in development before executions of bootstrap.sh
-set -e
+# WARNING: You take full resposibility of what could happen to your host when executing
+#          this script. This script has only been tested on Armen's MBP arm64 machine
+#
+# This script tries to remove as much as reasonable to allow for re-testing
+# bootstrap.sh multiple times on the same host
 
-# brew's uninstall script does not properly remove casks
-command -v brew &>/dev/null && brew uninstall --cask docker
-[ -d /Applications/Docker.app ] && rm -rf /Applications/Docker.app
+# Remove Sentry and dependencies
+rm -rf ~/.sentry
+rm -rf ~/code/sentry/{.venv,node_modules}
+rm -rf ~/code/getsentry/{.venv,node_modules}
+[ -f ~/.zprofile ] && rm ~/.zprofile
 
-# Since we install pre-commit via brew, we need to restore the hooks
-[ -f .git/hooks/pre-commit ] && pre-commit uninstall
+remove_brew_setup() {
```

Comment:
This does a more thorough clean-up than just calling the official uninstall script.